models/erc4337/arbitrum/erc4337_arbitrum_v0_5_userops_basics.sql

```diff
@@ -1,6 +1,7 @@
 {{ config
 (
-    alias ='v0_5_userops_basics',
+    schema = 'erc4337_v0_5_arbitrum',
+    alias ='userops_v0_5_basics',
```

Comment:
Hi @0xbitfly,
I think you may break some schema settings in the config blocks in the last commit.
This view now becomes `erc4337_v0_5_arbitrum.userops_v0_5_basics`.

Reply:
And the same goes to other changes in this commit.

Reply:
The table name is consists of `schema` + `.` + `alias`, this needs to be the same as `name` defined in schema.yml, and table name in ref(). Current changes may result in data loss/null pointer to tables in roll-up spells.